ccdproc/image_collection.py

```diff
@@ -112,6 +112,9 @@ def __init__(self, location=None, keywords=None, info_file=None,
         if keywords:
             self.keywords = keywords
 
+        if extension is not None:
```

Comment:
@MSeifert04 Unfortunately the error still persists.
Any reason you can think of?